bsunzip.sh

```diff
@@ -59,9 +59,9 @@ strip_cruft() {
 
 }
 
-# brightspace user comments are put in a quite unusable HTLM file
+# brightspace user comments are put in a quite unusable HTML file
 getcomment() {
-	tr -d '\n' < "$DEST"/index.html | sed 's/<tr bgcolor=#AAAAAA>/\n&/g' | grep -F "$1" | grep -F "$2"
+	tr -d '\n' < "$DEST"/index.html | sed 's/<tr bgcolor=#AAAAAA>/\n&/g' | perl -MHTML::Entities -pe 'decode_entities($_);' | grep -F "$1" | grep -F "$2"
```

Comment:
Did you add `perl` to the dependency check? Is `HTML::Entities` part of Perl's stdlib?

Reply:
no for the same reason we don't add sed or grep to the deps. Every sane system has perl because a lot of system tools use it.

I'm not sure whether `HTML::Entities` is in perl by default, I'll check this and add a check if this is not the case.

Reply:
`sed` and `grep` are POSIX requirements afaik, Perl is not. It's not installed on Debian or Arch by default.

Reply:
Yes, we don't need to , doesn't this hold for `unzip`? or `tar`? (I know that for some obscure reason modern POSIX wants us to ditch `tar in favour of `pax`, but I just don't see that going to happen.)

I would like to pick some arbitrary reference here. I am going to install a Debian netinst minimal image right now and see what tools it has.

Reply:
Note: without selecting 'install standard system utilities' (which already installs mutt...)

Reply:
And yes, that does have perl.

Reply:
Continued into PR #60.